server/methods/alexaApiCall.js

```diff
@@ -0,0 +1,23 @@
+import { Meteor } from 'meteor/meteor';
+import { HTTP } from 'meteor/http';
+
+import { settings } from '../../app/settings';
+
+Meteor.methods({
+	registerAlexaUser(serverurl, servername, userid, token) {
+		const apiUrl = settings.get('Register_Alexa_Enable_Server_Proxy_URL');
```

Comment:
Validate the logged user and use the if from there (the session) instead of passing it via parameter.

Reply:
> Validate the logged user and use the if from there (the session) instead of passing it via parameter.

I need to pass it as a parameter because it is then send to Proxy side as body param.

Reply:
@PrajvalRaval you should not pass the userId, you can get it calling `Meteor.userId()` here instead of calling it on the client-side and pss it via parameter.

Reply:
That way you ensure that you are getting the id of the logged user rather than believe in what the caller passed preventing someone to discover the id of another user and pass it manually.

Reply:
@rodrigok Fixed the userid parameter problem, now its only being called on the server side as you suggested. Let me know if I need to modify anything else 👍